modules/luci-mod-freifunk/root/usr/share/rpcd/acl.d/luci-mod-freifunk.json

```diff
@@ -0,0 +1,11 @@
+{
+	"luci-mod-freifunk": {
+		"description": "Provdies access to freifunk",
+		"read": {
+			"uci": [ "freifunk", "profile_*" ],
+		},
+		"write": {
+			"uci": [ "freifunk", "profile_*" ],
```

Comment:
I don't think, write access to the profiles_* is needed, as these files are only needed to look up some defaults.

Reply:
Wow, "profile (Expert)" - I always thought this is read-only ... So write-permission are required.

Even I'm not sure if it's a great idea to edit a community profile, but that's another issue.